websocket.go

```diff
@@ -344,7 +344,7 @@ func (wc *WebSocketClient) Subscribe(channels ...*WebSocketSubscribeMessage) err
 		case err := <-wc.errors:
 			return errors.Errorf("Subscribe failed, %s", err.Error())
 		case <-time.After(wc.timeout):
-			return errors.Errorf("Wait ack message timeout in %d s", 5)
+			return errors.Errorf("Wait ack message timeout in %v", wc.timeout)
```

Comment:
It is better to add time unit (s)

Reply:
It's added by default. Checkout [this](https://play.golang.org/p/8caHzQHwaVl) example.

Reply:
> It's added by default. Checkout [this](https://play.golang.org/p/8caHzQHwaVl) example.

Perfect!